Library/Homebrew/cmd/info.rb

```diff
@@ -239,7 +239,7 @@ def info_formula(f, args:)
 
     if stable = f.stable
       s = "stable #{stable.version}"
-      s += " (bottled)" if stable.bottled?
+      s += " (bottled)" if stable.bottled? and f.pour_bottle?
```

Comment:
@maxim-belkin `brew style --fix` for this.

Reply:
oh... thanks :)